docs/components/Code.tsx

```diff
@@ -104,6 +108,11 @@ function LiveCode({
 
 	return (
 		<Box border rounded borderColor="muted" className={proseBlockClassname}>
```

Comment:
Should this be our `Card`? It's probably (and should be) the same visual output, but code-wise it seems a bit cleaner.